File: app/src/main/java/com/digicoffer/lauditor/Email/AttachmentModel.java
package com.digicoffer.lauditor.Email;

import java.util.List;

public class AttachmentModel {
    String partId;
    String mimeType;
    String filename;
    int Size;
    String Id;

    public int getSize() {
        return Size;
    }

    public void setSize(int size) {
        Size = size;
    }

    public String getId() {
        return Id;
    }

    public void setId(String id) {
        Id = id;
    }

    public String getContentType() {
        return ContentType;
    }

    public void setContentType(String contentType) {
        ContentType = contentType;
    }

    String ContentType;
    // Add more fields if needed to represent attachment data

    public String getPartId() {
        return partId;
    }

    public void setPartId(String partId) {
        this.partId = partId;
    }

    public String getMimeType() {
        return mimeType;
    }

    public void setMimeType(String mimeType) {
        this.mimeType = mimeType;
    }

    public String getFilename() {
        return filename;
    }

    public void setFilename(String filename) {
        this.filename = filename;
    }

    public void setHeaders(List<Header> headers) {
    }

    public void setBody(Body body) {
    }
}

File: app/src/main/java/com/digicoffer/lauditor/Email/AttachmentModel.kt
package com.digicoffer.lauditor.Email

class AttachmentModel {
    @JvmField
    var partId: String? = null
    fun getPartId(): String? = partId
    fun setPartId(partId: String?) {
        this.partId = partId
    }

    @JvmField
    var mimeType: String? = null
    fun getMimeType(): String? = mimeType
    fun setMimeType(mimeType: String?) {
        this.mimeType = mimeType
    }

    @JvmField
    var filename: String? = null
    fun getFilename(): String? = filename
    fun setFilename(filename: String?) {
        this.filename = filename
    }

    @JvmField
    var size: Int = 0
    fun getSize(): Int = size
    fun setSize(size: Int) {
        this.size = size
    }

    @JvmField
    var id: String? = null
    fun getId(): String? = id
    fun setId(id: String?) {
        this.id = id
    }

    @JvmField
    var contentType: String? = null
    fun getContentType(): String? = contentType
    fun setContentType(contentType: String?) {
        this.contentType = contentType
    }

    fun setHeaders(headers: List<Header>?) {}
    fun setBody(body: Body?) {}
}
